PowerShell/JumpCloud Module/Public/Authentication/Connect-JCOnline.ps1

```diff
@@ -154,6 +154,7 @@ Function Connect-JCOnline ()
                 # Each time a new org is selected get settings info
                 $global:JCSettingsUrl = $JCUrlBasePath + '/api/settings'
                 $global:JCSettings = Invoke-JCApi -Method:('GET') -Url:($JCSettingsUrl)
+                $global:JCOrgSettings = (Get-JcSdkOrganization -Id $env:JCOrgId).Settings
```

Comment:
Saving these settings using a global variable instead of the initial env variable due to datatype issues